fdbrpc/Net2FileSystem.cpp

```diff
@@ -115,6 +115,10 @@ Net2FileSystem::Net2FileSystem(double ioTimeout, std::string fileSystemPath)
 #endif
 }
 
+Future<Void> Net2FileSystem::renameFile(const std::string &from, const std::string &to) {
+	return Void();
```

Comment:
Should this not be the actual implementation?

Reply:
ha -- I missed to replace this dummy code, good catch!